### ISSUE TITLE
ol style in 3d does not change on when style function changes .

### ISSUE THREAD

**Issue by arvindshekar07** [openlayers/ol-cesium#667]
Hi I am trying to update the styles of geojson on  some toggle action. 

```
       import { Stroke, Style } from 'ol/style';
           public updateLayerStyles(styles: Map<string, Style>,geoJsonLayer:VectorLayer): void {
            const withData = (feature: Feature) => {
                return styles.get(feature.getProperties().type);
            }
            geoJsonLayer.setStyle(withData);
      }
```
The above function works well with openlayer 2d map  but I dont see the changes getting reflected to 3d map .

example only show how to change as with a style object and not with stylefunction.

**Comment by gberaudo** [comment]
Yes, there is no way for OL-Cesium to know the function is returning a different result.
You can call `changed()` on the particular feature that changed.

**Comment by arvindshekar07** [comment]
So your saying that I could call changed() for feature wise change or create a new layer attaching the new style function?

**Comment by arvindshekar07** [comment]
thanks for the help. I think I shall change the vector layer in that case . I was hoping any change to  setStyle() would retrigger the resynchronization of the layer . If there that not a possibility then I'll close the issue.
Thanks.

**Comment by mehdiElma** [comment]
Hi @gberaudo , I'm facing the same issue as @arvindshekar07. Indeed, my data came from WFS geoserver and when I try to apply a style on these data, in 3D mode, nothing happen. I can't figure out how to trigger a re-synchronization of the feature to Cesium counterpart. Is there a solution implemented for this issue ? Can you help me please ? 

Regards.
